02-oop-inheritance/Module pattern/script.js

```diff
@@ -0,0 +1,47 @@
+
+var inheritPrototype = function (childClass, parentClass){
+
+    var copyOfParent = Object.create(parentClass.prototype);
+    copyOfParent.constructor = childClass;
+    childClass.prototype = copyOfParent;
+};
+
+var Movie = (function () {
```

Comment:
I thought that was the correct way to do it. So is it ok if i use a regular function?